### ISSUE TITLE
[CLI]: Histograms of single-item arrays are blank

### ISSUE THREAD

**Issue by zplizzi** [wandb/client#3601]
### Describe the bug

It seems like histograms of single-item arrays don't work. I would expect them to display as a single line/bar at the value of that item, but instead in the WandB UI the histogram display is just blank. I think the data is recorded properly, because the axis bounds seem about right. And if I hover over the graph I see a tooltip that has some blue bars. So maybe this is a UI bug and not a client bug.

This happens for manually-recorded histograms, and also those generated by wandb.watch() (eg for a single-element PyTorch Parameter).

Let me know if you can't reproduce this and I can help. 

### Environment

WandB version: 0.12.5

OS: Ubuntu 20.04

Python version:

Versions of relevant libraries:

**Comment by zplizzi** [comment]
Sure, here's a screenshot of a parameter histogram generated by wandb.watch(). The parameter has a single value (tensor of shape (1, )). 

<img width="445" alt="image" src="https://user-images.githubusercontent.com/5598968/166475647-34798e42-7c52-4135-960c-6611de159839.png">


Here's the (private) run this came from: https://wandb.ai/sourceress/zack_test/runs/1urfz7lp. You can see it in the parameters section.

**Comment by ash0ts** [comment]
​Hi @zplizzi ,

We wanted to follow up with you regarding your support request as we have not heard back from you. Please let us know if we can be of further assistance or if your issue has been resolved.
​

**Comment by zplizzi** [comment]
Ok, here's a script reproducing the issue i'm seeing (tested on wandb 0.12.16):

```
import torch
import wandb
from torch import nn


class Test(nn.Module):
    def __init__(self):
        super().__init__()
        self.a = nn.Linear(10, 1)

    def forward(self, x):
        return self.a(x)


if __name__ == "__main__":
    wandb.init(project="zack_test")
    model = Test()
    wandb.watch(model, log="all", log_freq=1, log_graph=False)
    opt = torch.optim.Adam(model.parameters())

    for i in range(100):
        opt.zero_grad()
        data = torch.zeros((5, 10))
        out = model(data)
        loss = out.sum()
        loss.backward()
        opt.step()
```

Here's what the output looks like:
<img width="918" alt="image" src="https://user-images.githubusercontent.com/5598968/167319243-c95d8f2a-2bf7-43d6-9e32-72a298a7c967.png">

**Comment by zplizzi** [comment]
In particular, the bias is a single-element tensor, so it should definitely be displayed in the `parameters/a.bias` chart.

**Comment by ash0ts** [comment]
Hello @zplizzi,

I was able to recreate and see what the issue you're talking about is. I have raised this with our engineers to take a look at. 

> I would expect them to display as a single line/bar at the value of that item.
Are there any other suggestions that you would also like to see implemented regarding this?

Thank you for bringing this to our attention

**Comment by ash0ts** [comment]
Hello @zplizzi,

Just wanted to check in to see if there any other suggestions that you would also like to see implemented regarding this.
Thank you!

**Comment by zplizzi** [comment]
nope, just fixing this would be great

**Comment by matekrk** [comment]
Hi there, 

Just to follow up to this issue. I believe it still has not been fixed for tensorboardX, couple of examples are attached below. 
![Zrzut ekranu z 2023-05-11 03-26-10](https://github.com/wandb/wandb/assets/25355486/e2e37611-9ea7-4cea-8b18-a1128505f344)
![Zrzut ekranu z 2023-05-11 03-25-22](https://github.com/wandb/wandb/assets/25355486/d638601d-4ccc-4509-85d7-5214a152f106)

**Comment by tomasff** [comment]
Hi, wondering if there's any update on this issue?

**Comment by ffarham** [comment]
Hi, I have the same issue when training a single parameter model.

<img width="415" alt="image" src="https://github.com/wandb/wandb/assets/55412683/68c3f08c-ad26-42e4-9641-3eff4ee6e650">
<img width="952" alt="image" src="https://github.com/wandb/wandb/assets/55412683/5c17b9b1-741d-4ce7-b3d3-49c62d12c648">

**Comment by sydholl** [comment]
WandB Internal User commented: 
ffarham commented: 
Hi, I have the same issue when training a single parameter model.

<img width="415" alt="image" src="https://github.com/wandb/wandb/assets/55412683/68c3f08c-ad26-42e4-9641-3eff4ee6e650">
<img width="952" alt="image" src="https://github.com/wandb/wandb/assets/55412683/5c17b9b1-741d-4ce7-b3d3-49c62d12c648">
